Source/Core/DolphinQt/Config/FreeLookWidget.cpp

```diff
@@ -35,6 +36,17 @@ void FreeLookWidget::CreateLayout()
   m_enable_freelook->SetDescription(
       tr("Allows manipulation of the in-game camera.<br><br><dolphin_emphasis>If unsure, "
          "leave this unchecked.</dolphin_emphasis>"));
+#ifdef USE_RETRO_ACHIEVEMENTS
+  bool hardcore = Config::Get(Config::RA_HARDCORE_ENABLED);
+  m_enable_freelook->setEnabled(!hardcore);
+  if (hardcore)
+  {
+    m_enable_freelook->SetDescription(
+        tr("<dolphin_emphasis>Disabled in Hardcore Mode.</dolphin_emphasis><br><br>Allows "
```

Comment:
I could see someone forgetting they have hardcode mode on.  I feel like being a little more wordy makes sense:

`<dolphin_emphasis>RA Hardcode Mode is enabled so this control is disabled.  To reenable, disable RA Hardcode Mode.</dolphin_emphasis>`

also not sure if the original description would make sense while Hardcode mode is active since they can't use it.

Reply:
Actually.  Can we use the `HardcoreWarningWidget` instead?

Reply:
Sure, that didn't exist the first time I made this change, I suppose I can add it now.